packages/core-p2p/package.json

```diff
@@ -33,7 +33,7 @@
         "@arkecosystem/crypto": "4.0.0-next-1",
         "@hapi/boom": "9.0.0",
```

Comment:
`core-p2p` and `core-webhooks` were not updated to `@hapi/boom@9.1.4` and pulled in older `@hapi/boom@9.0.0` due to strict version match (using `9.0.0` not `^9.0.0`).